static/skin/index.js

```diff
@@ -279,36 +279,24 @@
         });
     }
 
+    function setNoResultsContent() {
+        const kiwixHomeBody = document.querySelector('.kiwixHomeBody');
+        const divTag = document.createElement('div');
+        divTag.setAttribute('class', 'noResults');
+        divTag.innerHTML = `No result. Would you like to <a href="/?lang=">reset filter</a>?`;
+        kiwixHomeBody.append(divTag);
+        kiwixHomeBody.setAttribute('style', 'display: flex; justify-content: center; align-items: center');
+        loader.setAttribute('style', 'position: absolute; top: 50%');
+    }
+
```

Comment:
The message of this commit must explain why this function is shorter than the inline code that it is supposed to replace. Or, even better, you can make this change in two steps:

1. Drop the `onclick` handler for the "reset filter" link (explaining why it was needed & what was wrong with that workaround)
2. Extract the `setNoResultsContent()` function

Reply:
Done, thank you!